CHANGELOG.md

```diff
@@ -47,19 +52,10 @@ This release is intended for the PureScript 0.9.1 compiler and newer.
 
 **Note**: The v1.0.0 tag is not meant to indicate the library is “finished”, the core libraries are all being bumped to this for the 0.9 compiler release so as to use semver more correctly.
 
-## [v1.0.0-rc.1](https://github.com/purescript/purescript-st/releases/tag/v1.0.0-rc.1) - 2016-03-16
-
-- Release candidate for the psc 0.8+ core libraries
-
 ## [v0.1.1](https://github.com/purescript/purescript-st/releases/tag/v0.1.1) - 2015-09-12
 
-Simplify the type of `pureST`.
+- Simplify the type of `pureST`.
 
 ## [v0.1.0](https://github.com/purescript/purescript-st/releases/tag/v0.1.0) - 2015-06-30
 
-This release works with versions 0.7.\* of the PureScript compiler. It will not work with older versions. If you are using an older version, you should require an older, compatible version of this library.
-
-## [v0.1.0-rc.1](https://github.com/purescript/purescript-st/releases/tag/v0.1.0-rc.1) - 2015-06-06
-
-Initial release candidate.
```

Comment:
I've removed the `rc` releases, as we don't usually include them in changelogs (only major, minor, and patch releases).